### ISSUE TITLE
Random error messages pop up

### ISSUE THREAD

**Issue by Arroyodorado** [destefanis/design-lint#48]
Hi there,

I'm very sorry not being able to be more specific but when using the plugin (which is great!) from time to time error toasts like these appear at the bottom of Figma's window:

![Bildschirmfoto 2023-02-06 um 12 57 35](https://user-images.githubusercontent.com/10092794/217043617-c482d375-e357-492a-b855-e1d946a09fa9.png)

When opening the console this can be read:

![Bildschirmfoto 2023-02-06 um 12 59 12](https://user-images.githubusercontent.com/10092794/217043700-64ecdf3a-8c4b-43f7-94d8-df825d366f91.png)

Afterwards, using other plugins is also likely to produce error messages of various kinds. Restarting the Figma app solves the problem (obviously).

I'm aware that all this is quite vage but I haven't found a pattern yet that causes all this. Can you read something useful from it nonetheless?

If you want me to send specific information, I'd be happy to help. As said before, the plugin is great!

**Comment by destefanis** [comment]
Yeah i'm not sure if there was a bug with the recent addition of sections and border thickness, but I think FIgma sometimes can run into bugs with running plugins. I haven't seen this one in some time so i'm going to mark this closed!
